styles/components/grid.css

```diff
@@ -0,0 +1,27 @@
+.grid {
+  display: grid;
+  width: 100%;
+  gap: 1rem;
+}
+
+.col-2 {
+  grid-template-columns: repeat(2, minmax(300px, 1fr));
+}
+
+.col-3 {
+  grid-template-columns: repeat(3, minmax(200px, 1fr));
+}
+
+.grid-col {
```

Comment:
advice:- you can create  a flex center helper class, and can use it  wherever you need flex or flex center

Reply:
Thanks for suggesting Himanshu but I am not using any utility classes, because I don't want to have multiple classes on a particular element. So that's why I am giving all these properties again and again